browser/ui/tabs/tab_features.cc

```diff
@@ -54,6 +60,20 @@ void TabFeatures::Init(TabInterface& tab, Profile* profile) {
     tab_data_observer_ = std::make_unique<ai_chat::TabDataWebContentsObserver>(
         tab.GetHandle().raw_value(), tab.GetContents());
   }
+
+#if BUILDFLAG(ENABLE_PSST)
+  psst_observer_ = psst::PsstTabWebContentsObserver::MaybeCreateForWebContents(
+      tab.GetContents(),
+      std::make_unique<psst::PsstDialogTabHelperDelegateImpl>(
+          tab.GetContents()),
+      ISOLATED_WORLD_ID_BRAVE_INTERNAL);
+#endif
+}
+
+#if BUILDFLAG(ENABLE_PSST)
+psst::PsstTabWebContentsObserver* TabFeatures::GetPsstTabHelper() {
```

Comment:
this should be an inline accessor in the header file with underscore naming, but also you're referring to this using three different names. The class is `PsstTabWebContentsObserver`, the method name is `GetPsstTabHelper` and the member name is `psst_observer_`. Please use consistent naming so `psst_web_contents_observer_` and `psst_web_contents_observer()`